SELF_HOSTED_INSTRUCTIONS.md

```diff
@@ -0,0 +1,48 @@
+## First one time set up
+Make sure you have running uwazi [dependencies](https://github.com/huridocs/uwazi#dependencies).
+
+First time installing uwazi we are going to create a basic folder structure to deploy the code and the folders that will contain potential uploaded or created files by uwazi.
```

Comment:
"During first time installation we create a basic folder structure that will contain potential uploads or files created by Uwazi, as well as the production build of the code"